ignite/cmd/plugin.go

```diff
@@ -44,15 +45,21 @@ func LoadPlugins(ctx context.Context, rootCmd *cobra.Command) error {
 	return loadPlugins(rootCmd, plugins)
 }
 
-func parseLocalPlugins(rootCmd *cobra.Command) (cfg *pluginsconfig.Config, path string, err error) {
-	appPath := flagGetPath(rootCmd)
-	pluginsPath, err := pluginsconfig.LocateDefault(appPath)
+func parseLocalPlugins(cmd *cobra.Command) (*pluginsconfig.Config, error) {
+	// FIXME(tb): like other commands that works on a chain directory,
+	// parseLocalPlugins should rely on `-p` flag to guess that chain directory.
+	// Unfortunately parseLocalPlugins is invoked before flags are parsed, so
+	// we cannot rely on `-p` flag. As a workaround, we use the working dir.
+	// The drawback is we cannot load chain's plugin when using `-p`.
+	_ = cmd
```

Comment:
An alternative would be to parse the "-p" (long flag name too) from `os.Args` in a custom function

Reply:
Yes, even if I don't like to duplicate cobra's job, that's probably what we are going to do finally, because this issue becomes more and more annoying.